deny.toml

```diff
@@ -20,9 +18,10 @@ skip = [
     { name = "hashbrown", version = "0.12.3" },
     { name = "hashbrown", version = "0.14.0" },
     { name = "nix", version = "0.26.4" },
-    { name = "nix", version = "0.27.1" },
```

Comment:
can we continue to add all versions of duplicate deps here? it makes it easier to track when things change

Reply:
we can but cargo deny was yelling at me, maybe I need to adjust my configuration?